tools/zbcat

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+# Contributed by Bryan Halfpap <Bryanhalf@gmail.com>, Copyright 2016
+import sys
+import argparse
+from time import sleep
+import logging
+from killerbee import *
+
+"""
+====================================
+ZBCAT - The 802.15.4 packet pusher!
+====================================
+Pushes 802.15.4 DATA packets onto the specified channel, to the specified PAN and Destination address from the defined source address.
+Can be configured to respond to DATA REQUEST packets (--respond) or it can just spew packets out to a target (maybe it wasn't requesting, but is still accepting)
+Written to help facilitate these situations due to a Wireless Village Wireless Capture-the-Flag at Shmoocon 2016 by @redbarondefcoin
+"""
+
+def create_packet(panid, source, dest, data, extsource=False, ack=False, shortaddress=True, seq=0):
+    if extsource == False:
+        extsource = ''
+
+    if shortaddress:
+        mode = 0b10
+    else:
+        mode = 0b11
+
+    # bit manipulating the FCS was the quickest way, but here I break down the bytes in the order they are pushed (starting with the left-most bit)
```

Comment:
This is something we keep debating how to solve best. Construction using the dot15d4 scapy library is the cleanest, and would give us something to standardize to. However, it adds a dependency, so I understand why you did it this way. What are your thoughts on this?

Reply:
@Crypt0s  - I think we should change this from saying FCS and using fcs as a variable to calling it FCF and using fcf as a variable. The FCS is the checksum at the end, but this is the Frame Control field.

Reply:
(I'll do this in the new PR from this branch down to master)